addons/survey/controllers/main.py

```diff
@@ -712,14 +713,139 @@ def _generate_report(self, user_input, download=True):
             ('Content-Disposition', report_content_disposition),
         ])
 
-    def _get_user_input_domain(self, survey, line_filter_domain, **post):
-        user_input_domain = ['&', ('test_entry', '=', False), ('survey_id', '=', survey.id)]
-        if line_filter_domain:
-            matching_line_ids = request.env['survey.user_input.line'].sudo().search(line_filter_domain).ids
-            user_input_domain = expression.AND([
-                [('user_input_line_ids', 'in', matching_line_ids)],
-                user_input_domain
-            ])
+    @classmethod
+    def _extract_filters_data(cls, request, survey_id, post):
+        """
+            Extracts the filters from the URL and returns the related
+            user_input_line_ids and filters data.
+        """
+
+        def _get_filter_items_matching_question_answers(answer_cols_to_rows_mapping):
+            filters_domain_items = []
+            filters_data = []
```

Comment:
I think we can keep the original name here.
```suggestion
            search_filters = []
```